data/dataloader.py

```diff
@@ -15,7 +15,7 @@
 
 
 try:
-    with open("../../data-path.txt", "r") as f:
+    with open("../data/", "r") as f:
```

Comment:
misunderstanding here i think. `data-path.txt` is a text file 2 folders above this code folder, which contains the path to the actual data. this lets me "inform" all my projects about changes to this path in a single update. 

to help understand: the structure on my computer is: `projects-folder/specific-project-folder/specific-project-code` . i keep `data-path.txt` in `projects-folder` , and data path contains an absolute path to local data on my computer in its first line, e.g. `/Users/gail/mydata`. if i ever need to change this path, i simply update the contents of `data-path.txt`, and all my projects are informed. 

so long as you don't need it, you don't need to define this file. the code will simply recognise that there are no personal datasets in this case.

Reply:
ahhhhh okay cool, I didn't understand it